.changeset/build-upgrade-lerna.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
```

Comment:
I know it isn't technically a docs change, but in order for our build to pass it needs a package here. If we use `docs`, our adopters won't see it in the changelog list which they don't need